service/controller/resource/monitoring/scrapeconfigs/test/aws/case-0-cluster-api.golden

```diff
@@ -1,1185 +0,0 @@
-- job_name: bob-prometheus/kubernetes-apiserver-bob/0
```

Comment:
why do all `case-0-cluster-api` tests disappear? Is it expected?

Reply:
I think those were leftovers because we do not have a case-0 input

Reply:
Makes sense, given CAPI tests are as separate cloudproviders. BTW, looks like we don't test all cloudproviders.

Reply:
Yes I think we need to rework those tests with the addition of CAPI